zigzag/zigzag_test_log.py

```diff
@@ -303,3 +303,27 @@ def _get_test_run_failure_output_field_id(cls, mediator):
             cls._test_run_failure_output_field_id = \
                 UtilityFacade(mediator).find_custom_field_id_by_label('Failure Output', 'test-runs')
         return cls._test_run_failure_output_field_id
+
+    @classmethod
+    def _get_fields(cls, mediator):
+        """Gets the fields from this class
+
+        Args:
+            mediator (ZigZag): The ZigZag mediator
+
+        Returns:
+            dict(str: ZigZagTestLogField)
+        """
+
+        if cls._fields == 0:
+            cls._fields = {}
+            for prop, value in list(mediator.testsuite_props.items()):
+                field = {
+                    'id': UtilityFacade(mediator).find_custom_field_id_by_label(prop, 'test-runs'),
+                    'name': prop,
+                    'value': value
+                }
+                if field['id']:
+                    cls._fields[prop] = ZigZagTestLogField(value)
```

Comment:
`ZigZagTestLogField` expects a kwargs as input, but you've provided a single argument.

Reply:
You could just write it as:

```python
cls._fields[prop] = ZigZagTestLogField(**field)
```